util/desensitize/desensitize.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package desensitize
```

Comment:
This is more of a product issue, but we should rename this from "desensitize" to 
"redact".

"desensitize" means to make *less* sensitive. So if we only remove some sensitive information it will be desensitized. Seeing this called desensitize won't give me confidence that it is thorough.
"redact" means to completely omit some information. Also I believe the terminology is more standard in the database community.

Reply:
Great, thanks~